src/render/react/core/reconciler/index.js

```diff
@@ -124,12 +123,16 @@ const HostConfig = {
   commitTextUpdate(textInstance, oldText, newText) {
     textInstance.setText(newText);
   },
+  detachDeletedInstance: (instance) => {
+    unRegistEvent(instance.uid);
+    instanceMap.delete(instance.uid);
+    instance.style = null; // Proxy preventing GC
```

Comment:
I realized that this line can sometimes cause a crash of the stress test. But without it, the GC just do not collect the detached component, which leads to a big memory leak. I need to investigate further how to solve that.

Reply:
Fixed in f86a19a